src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx

```diff
@@ -2,3 +2,243 @@ export const meta = {
   title: `Connect your app code to the API`,
   description: `Learn how to connect your app code to the API.`
 };
+
+import guide from "/src/fragments/common/guide-header.mdx";
+
+<Fragments fragments={{all: guide}} />
+
+In this guide, you will connect your application code to the backend API using the Amplify Libraries. We will review how to install and configure Amplify Libraries.
+
+Before you begin, you will need:
+- An GraphQL API already set up either via Amplify or via the underlying AWS AppSync service. Review [TK]() on how to build your API and database with Amplify. For guidance on how to build a GraphQL API via AWS AppSync, see [Designing a GraphQL API in AWS AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html).
+- A frontend application set up. For instructions on setting up an app, see [Project Setup: Create your application](https://docs.amplify.aws/lib/project-setup/create-application/q/platform/js/)
+- [npm installed](https://docs.npmjs.com/getting-started)
+
+## Install and configure the Amplify Libraries
+
+Install the `aws-amplify` NPM package to your app. This is the main library for working with Amplify Libraries in your projects. This will include connecting your app with the GraphQL endpoint. 
+
+If you are using `Next.js`, `Vue`, or `Angular`, please review the following details for installing the Amplify Libraries:
```

Comment:
@renebrandel Line 37 "After reading to the bottom, I'm not sure if this makes sense to be here. I think we need to do some consolidation with the bottom blocks. Let's discuss on a call to align on the edits." 

I updated this section and moved the import config details into the block switcher below so place to start editing from. However, I did not add the yarn info b/c all switchers focus currently on npm. Also updated the order in text and switcher.